crates/db_schema/src/impls/comment.rs

```diff
@@ -139,14 +86,71 @@ where ca.comment_id = c.id",
       .get_result::<Self>(conn)
   }
 
-  pub fn upsert(conn: &PgConnection, comment_form: &CommentForm) -> Result<Comment, Error> {
+  pub fn create(
```

Comment:
Should still be called upsert as thats what it does. Maybe consider having separate public methods for create/insert, and a private method which does the ltree related work.

Reply:
That'd mean that every Comment::create needs to be changed to comment::upsert, since they can and should do the same thing.